python/labbox_ephys/helpers/prepare_snippets_h5.py

```diff
@@ -101,3 +113,41 @@ def prepare_snippets_h5_from_extractors(
             f.create_dataset(f'unit_waveforms/{unit_id}/channel_ids', data=np.array(channel_ids_by_unit[int(unit_id)]).astype(int))
             f.create_dataset(f'unit_waveforms/{unit_id}/spike_train', data=np.array(sorting_subsampled.get_unit_spike_train(unit_id=unit_id)).astype(np.float64))
             
+def prepare_snippets_nwb_from_extractors(
+    recording: se.RecordingExtractor,
+    sorting: se.SortingExtractor,
+    nwb_file_path: str,
+    nwb_object_prefix: str,
+    start_frame,
+    end_frame,
+    max_neighborhood_size: int,
+    max_events_per_unit: Union[None, int]=None,
+    snippet_len=(50, 80),
```

Comment:
```suggestion
    recording: se.RecordingExtractor,
    sorting: se.SortingExtractor,
    nwb_file_path: str,
    is_noise_sorting: bool,
    start_frame,
    end_frame,
    max_neighborhood_size: int,
    max_events_per_unit: Union[None, int]=None,
    snippet_len=(50, 80),
    sort_interval=None,
    sort_interval_valid_times=None,
    templates=None
```
The change below requires a change to the function header.